packages/backend/src/graphql/resolvers/auto-peering.test.ts

```diff
@@ -244,85 +235,90 @@ describe('Auto Peering Resolvers', (): void => {
       ${AutoPeeringError.InvalidPeerUrl}
       ${AutoPeeringError.InvalidPeeringRequest}
       ${AutoPeeringError.LiquidityError}
-    `('4XX - $error', async ({ error }): Promise<void> => {
+    `('Errors with $error', async ({ error: testError }): Promise<void> => {
       jest
         .spyOn(autoPeeringService, 'initiatePeeringRequest')
-        .mockResolvedValueOnce(error)
+        .mockResolvedValueOnce(testError)
       const input = createOrUpdatePeerByUrlInput()
-      const response = await appContainer.apolloClient
-        .mutate({
-          mutation: gql`
-            mutation CreateOrUpdatePeerByUrl(
-              $input: CreateOrUpdatePeerByUrlInput!
-            ) {
-              createOrUpdatePeerByUrl(input: $input) {
-                code
-                success
-                message
-                peer {
-                  id
+      try {
```

Comment:
```suggestion
      expect.assertions(2)
      try {
```
to make sure this actually throws and the test does the correct number of assertions (same comment throughout)